Error with mypy event in master branch

When I try to run mypy on khal project I've got this error 
```
tests/ui/test_calendarwidget.py:6: error: Need type annotation for 'on_press' (hint: "on_press: Dict[<type>, <type>] = ...")
khal/khalendar/backend.py:190: error: Incompatible default for argument "stuple" (default has type "object", argument has type "Tuple[Any, ...]")
tests/cli_test.py:405: error: Not all arguments converted during string formatting
```

For the `backend.py` file, the source of problem is `sql_ex()`, the `stuple: tuple=Union[tuple, str]` parameter. Removing the `tuple=` part avoid the problem.

--- a/khal/khalendar/backend.py
+++ b/khal/khalendar/backend.py
@@ -28,7 +28,7 @@ from enum import IntEnum
 import logging
 import sqlite3
 from os import makedirs, path
-from typing import Any, Dict, Iterable, List, Optional, Tuple, Union
+from typing import Any, Dict, Iterable, List, Optional, Tuple
 
 import icalendar
 import pytz
@@ -187,7 +187,7 @@ class SQLiteDb(object):
                 stuple = (cal, '')
                 self.sql_ex(sql_s, stuple)
 
-    def sql_ex(self, statement: str, stuple: tuple=Union[tuple, str]) -> List:
+    def sql_ex(self, statement: str, stuple: tuple) -> List:
         """wrapper for sql statements, does a "fetchall" """
         self.cursor.execute(statement, stuple)
         result = self.cursor.fetchall()

